bears/go/GoTypeBear.py

```diff
@@ -23,4 +23,4 @@ class GoTypeBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '-e', filename
+        return ('-e', filename)
```

Comment:
I was getting error in this line, so was just checking if anything changes by this. Though, it doesn't affect the code.